guides/source/asset_pipeline.md

```diff
@@ -656,14 +656,27 @@ JS/CSS is excluded, as well as raw JS/CSS files; for example, `.coffee` and
 `.scss` files are **not** automatically included as they compile to JS/CSS.
 
 If you have other manifests or individual stylesheets and JavaScript files to
-include, you can add them to the `precompile` array in `config/initializers/assets.rb`:
+include, you can add them yourself:
 
-```ruby
-Rails.application.config.assets.precompile += %w( admin.js admin.css )
-```
+- **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:
+
+  ```ruby
+  Rails.application.config.assets.precompile += %w( admin.js admin.css )
+  ```
+
+- **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`
+
+  ```js
+  //= link_tree ../images
+  //= link_directory ../javascripts .js
+  //= link_directory ../stylesheets .css
+  ... (other files/directories)
+  ```
```

Comment:
I think it is clearer if we use the same admin assets in the example:
```suggestion
  ```js
  //= link_tree ../images
  //= link admin.js
  //= link admin.css
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
  ```
```